.github/workflows/lint.yml

```diff
@@ -16,9 +16,15 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.7
-      - name: Install Lint Dependencies
-        run: pip install pyflakes==2.1.1 black==19.10b0
-      - name: Run PyFlakes
-        run: pyflakes .
+      - name: Install project dependencies
+        run: |
+          pip install tensorflow==2.0.0 colorama==0.4.1
+          pip install -e .[test]
+      - name: Run flake8
+        run: flake8
       - name: Black code style
-        run: black . --check --target-version py36 --exclude 'build/|buck-out/|dzzist/|_build/|\.git/|\.hg/|\.mypy_cache/|\.tox/|\.venv/|larq/snapshots/'
```

Comment:
Maybe we should keep `--target-version py36`

Reply:
See above

Reply:
For black we can still use `--target-version py36` since it doesn't relate to the current interpreter version. It just makes sure to format the code to be compliant with Python 3.6. In most cases this doesn't matter, but there are some edge cases like trailing commas that can change formatting.